Src/Support/Google.Apis.Auth/OAuth2/GoogleCredential.cs

```diff
@@ -326,6 +326,45 @@ void IConfigurableHttpClientInitializer.Initialize(ConfigurableHttpClient httpCl
         /// </summary>
         public ICredential UnderlyingCredential => credential;
 
+        /// <summary>Creates a <c>GoogleCredential</c> wrapping a <see cref="ImpersonatedCredential"/>.</summary>
+        /// <param name="TargetPrincipal">The service account to impersonate.</param>
+        /// <param name="Delegates">The chained list of delegates.</param>
+        /// <param name="Scopes">The scopes to request during the authorization grant.</param>
+        /// <param name="LifetimeInSeconds">
+        /// The number of seconds the delegated credential should be valid.
+        /// By default this value should be at most 3600. However, you can follow
+        /// https://cloud.google.com/iam/docs/creating-short-lived-service-account-credentials#sa-credentials-oauth
+        /// to set up the service account and extend the maximum lifetime to 43200 (12 hours).
+        /// If the given lifetime is 0, default value 3600 will be used instead when creating the credentials.
+        /// </param>
+        public GoogleCredential Impersonate(string TargetPrincipal, IEnumerable<string> Delegates = null, 
```

Comment:
All of these parameters should be camelCased, so `targetPrincipal`, `delegates`, `scopes` and `lifetimeInSeconds`

Reply:
done